docs/dashboards/cloud-posture.asciidoc

```diff
@@ -23,13 +23,9 @@ The Cloud Posture dashboard shows:
 [[cloud-posture-dashboard-UI]]
 === Cloud Posture dashboard UI
 
-The first row of cards (Cloud Posture Score, Failed Findings, and Open Cases) summarizes your overall cloud security posture (CSP) by aggregating data from all monitored Kubernetes clusters. Each subsequent row summarizes the CSP of an individual Kubernetes cluster.
+The top section of the dashboard summarizes your overall cloud security posture (CSP) by aggregating data from all monitored Kubernetes clusters. Three summary cards on the left of this section show Clusters Evaluated, Resources Evaluated, and Failed Findings. You can click *Resources Evaluated* or *Failed Findings* to open the <<findings-page, Findings page>> grouped by resources or filtered for failed findings, respectively. The remaining summary cards show your overall compliance score, and total failed findings grouped by CIS section. Click any CIS section name to view failed findings from that section on the <<findings-page, Findings page>>.
 
-The Cloud Posture Score card shows the collective performance of all your monitored clusters. Hover over the card to display when the data was collected.
-
-The Failed Findings card shows failed findings grouped by Center for Internet Security (CIS) benchmark categories. Click any section name to view its failed findings on the <<findings-page, Findings page>>.
-
-Each row of cards after the first summarizes an individual cluster's CSP. Each row includes the cluster's name, the last 6 digits of its ID, its compliance score, and failed findings grouped by CIS section. When you begin to monitor a new cluster, a new row of cards appears.
+Below the summary section, each row shows the CSP of an individual Kubernetes cluster, including the cluster's name, when its posture was last evaluated, its compliance score, and failed findings grouped by CIS section. When you begin to monitor a new cluster, a new row appears.
```

Comment:
What do we mean by "shows the CSP of an individual Kubernetes cluster..." are we saying it shows the cloud security posture of an individual...? Am I understanding it correctly? 
```suggestion
Below the summary section, each row shows the CSP of an individual Kubernetes cluster, including its name, when its posture was last evaluated, its compliance score, and failed findings grouped by CIS section. When you begin to monitor a new cluster, a new row appears.
```

Reply:
It means each row after the summary section just shows the findings from one cluster. Maybe "single" would be better than "individual"?